app/src/main/java/tech/relaycorp/courier/data/model/StorageUsage.kt

```diff
@@ -0,0 +1,10 @@
+package tech.relaycorp.courier.data.model
+
+import kotlin.math.roundToInt
+
+data class StorageUsage(
+    val used: Long,
+    val totalAvailable: Long
+) {
+    val percentage get() = ((used.toFloat() / totalAvailable) * 100).roundToInt()
```

Comment:
We'd need to make sure that any UI component using this result will work with a percentage greater than 100. If we're just displaying a number, that'd be fine. But if we're displaying a chart, for example, that'd be problematic.

Reply:
I'm displaying a bar for now, and I can make sure it doesn't break with values about 100%.